module/importer/src/Infrastructure/Handler/ImportVariableProductCommandHandler.php

```diff
@@ -28,14 +29,24 @@ class ImportVariableProductCommandHandler
      */
     private ImportErrorRepositoryInterface $repository;
 
+    /**
+     * @var LoggerInterface
+     */
+    private LoggerInterface $importLogger;
+
     /**
      * @param VariableProductImportAction    $action
      * @param ImportErrorRepositoryInterface $repository
+     * @param LoggerInterface                $importLogger
      */
-    public function __construct(VariableProductImportAction $action, ImportErrorRepositoryInterface $repository)
-    {
+    public function __construct(
+        VariableProductImportAction $action,
+        ImportErrorRepositoryInterface $repository,
+        LoggerInterface $importLogger
```

Comment:
Don't really get why logger is importLogger but repository i "just" repository. I think not specified properties names would suffice(so just "logger") but for sure it's good to be consistent and keep one approach.

Reply:
because  is specific import channel https://symfony.com/doc/current/logging/channels_handlers.html#how-to-autowire-logger-channels

Reply:
IMHO this sucks very much since it's directly coupling this code with the DI and SF autowiring approach - not very DDD. If that's the case for the product well, it is how it is...